src/Donations/Components/ContactsForm.tsx

```diff
@@ -459,6 +464,12 @@ function ContactsForm(): ReactElement {
                     label={t("companyName")}
                     variant="outlined"
                     data-test-id="test-companyname"
+                    inputProps={{ readOnly: profile !== null }}
```

Comment:
If it is a readOnly field when companyName is exists, maybe we should disable it. Just like how we are doing it for email field

<img width="887" alt="Screenshot 2023-02-23 at 11 34 29 AM" src="https://user-images.githubusercontent.com/37466979/220832755-ebc6579e-693b-41e0-987d-d47c1cc0d126.png">

Reply:
Done here [0aca0e4](https://github.com/Plant-for-the-Planet-org/planet-donations/pull/388/commits/0aca0e4fe3952c598d5a96923cb5abcaf1db5c6c)